scripts/rmc

```diff
@@ -25,8 +25,10 @@ def main():
     parser.add_argument("--allow-cbmc-verification-failure", action="store_true")
     parser.add_argument("--mangler", default="v0")
     parser.add_argument("--visualize", action="store_true")
+    parser.add_argument("--c-lib", nargs="+", default=[])
     parser.add_argument("--srcdir", default=".")
     parser.add_argument("--wkdir", default=".")
+    parser.add_argument("--end-rmc-flags", action="store_true")
```

Comment:
What does this do?

Reply:
There's a lot of confusing behavior with flags that take a non-set number of arguments. If you wanted to, e.g. just pass the `--c-lib` flag, it might look like `rmc --c-lib a.c b.c x.rs`, in which case the parser gets confused and thinks `x.rs` is part of the `--c-lib` flag, and it gets a parsing error. You can use `--` to denote the end, like `rmc --c-lib a.c b.c -- x.rs`, but then that runs into issues with starting the cbmc flags with `--`, since using `--` in two places causes issues. Having the `--end-rmc-flags` flag makes it easier to guarantee a well-formed command.

Reply:
Can we just make c lib able to be called multiple times? 
``` 
rmc --clib a.c --clib b.c
```